### DISCUSSION TITLE
接口配置

### DISCUSSION THREAD

**Discussion by zrgtester** [baidu/dperf#433] [General]
如果网口对端接的是二层带vlan或者三层子接口，dperf应该如何配置来支持vlan呢

**Comment by pengjianzhang** [comment]
请参考配置手册
vlan

**Comment by zrgtester** [comment]
是这个手册吗？https://dperf.org/doc/html/configuration
里面没有vlan相关的介绍。

**Comment by pengjianzhang** [comment]
看起来是配置手册里漏了，谢谢

配置文件里加一行，如
vlan   666
